lang/cs/Org.Apache.REEF.All/Org.Apache.REEF.All.csproj

```diff
@@ -34,45 +34,10 @@ under the License.
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
All those referenced csproj file need to stay by design. REEF.all is to wrap those dlls with no real code for clients.